patches/corefx/0004-wip.patch

```diff
@@ -0,0 +1,31 @@
+From c68184f74e3c8991ae90dec3414d3e7cc4eb4bf7 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Sat, 17 Nov 2018 16:34:00 -0600
+Subject: [PATCH] wip
+
+---
+ eng/common/tools.sh | 8 --------
+ 1 file changed, 8 deletions(-)
+
+diff --git a/eng/common/tools.sh b/eng/common/tools.sh
+index c7d03ee028..9aaae295bc 100644
+--- a/eng/common/tools.sh
++++ b/eng/common/tools.sh
```

Comment:
Just a note: `eng/common/**` is auto-updated from Arcade, so PR will have to go there instead of CoreFX.

Reply:
I removed this patch because we set `PACKAGES_DIR`: the removed block shouldn't have any effect anyway.